src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx

```diff
@@ -45,7 +45,7 @@ class DisabilityBenefitsApp extends React.Component {
     const { available, synced, authorized } = this.props;
 
     return (
-      <RequiredLoginView authRequired={3} serviceRequired={"disability-benefits"}>
+      <RequiredLoginView authRequired={3} serviceRequired={"evss-claims"}>
```

Comment:
We made the backend change to change this service name, too? I ask because I don't know how that works and assume it's separate from changing the api routes.